src/managing-deis/platform-ssl.md

```diff
@@ -1,3 +1,62 @@
 # Platform SSL
 
-TODO (bacongobbler): rewrite for v2
+SSL/TLS is the standard security technology for establishing an encrypted link between a web server
+and a browser. This link ensures that all data passed between the web server and browsers remain
+private and integral.
+
+To enable SSL for your cluster and all apps running upon it, you can add an SSL key to your load
+balancer. You must either provide an SSL certificate that was registered with a CA or provide
+[your own self-signed SSL certificate](../reference-guide/creating_self_signed_ssl.md).
+
+## Installing SSL on a Load Balancer
+
+On most cloud-based load balancers, you can install a SSL certificate onto the load balancer
+itself. Any communication inbound to the load balancer will be encrypted while the internal
+components of Deis will still communicate over HTTP.
+
+When you install Deis, Kubernetes will provision a load balancer for the routers. To enable SSL,
+you will need to modify the listener settings on the load balancer:
```

Comment:
This is only true on certain underlying infrastructure.  e.g. true on AWS and GKE, but valse on bare metal or vagrant.  It is possibly a good idea to mention that and to also mention that if running on a platform where the load balancer was not automatically provisioned by k8s, generic, manual load balancer configuration instructions are available here: https://github.com/deis/router#manually-configuring-a-load-balancer

Your doc here could instruct readers to resume these instructions after they have set up their load balancer.